e2e/workspace-create/src/create-nx-workspace-npm.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+  checkFilesDoNotExist,
+  checkFilesExist,
+  cleanupProject,
+  newProject,
+  runCLI,
+  uniq,
+} from '@nrwl/e2e/utils';
+
+describe('create-nx-workspace --preset=npm', () => {
+  process.env.SELECTED_PM = 'npm';
```

Comment:
Does this only work on `npm`? It's a bit weirdly named but the `--preset npm` should work with any package manager. The `npm` preset refers to the repo being geared toward publishing TO `npm`, not that it uses `npm` as a package manager.

Reply:
i ran into an issue in ci with pnpm.

Reply:
here is the run with error: https://app.circleci.com/pipelines/github/nrwl/nx/27038/workflows/7946dfd0-8fee-49b7-92d7-56cd8f049f3e/jobs/272993.

```
Command failed: pnpm add -D @nrwl/angular@16.0.0 @nrwl/eslint-plugin-nx@16.0.0 @nrwl/express@16.0.0 @nrwl/esbuild@16.0.0 @nrwl/jest@16.0.0 @nrwl/js@16.0.0 @nrwl/linter@16.0.0 @nrwl/nest@16.0.0 @nrwl/next@16.0.0 @nrwl/node@16.0.0 @nrwl/nx-plugin@16.0.0 @nrwl/rollup@16.0.0 @nrwl/react@16.0.0 @nrwl/storybook@16.0.0 @nrwl/vite@16.0.0 @nrwl/web@16.0.0 @nrwl/webpack@16.0.0 @nrwl/react-native@16.0.0 @nrwl/expo@16.0.0
```

Reply:
I think we should have this cross-package manager and investigate why those tests are failing.